src/components/decks/DeckListing.vue

```diff
@@ -59,11 +62,13 @@ export default {
     // This is the list of decks currently shown
     decks: null,
     deckCount: 0,
+    preconOnly: false
```

Comment:
I wasn't going to bother asking because it's super minor, but since there's a minor change anyway...would you mind adding trailing commas to this line and `Toggle` below? Just makes git diffs a little cleaner if/when we add more components and such. 🙂